ui/app/com/gu/recipeasy/controllers/Application.scala

```diff
@@ -18,6 +18,16 @@ import models.CuratedRecipeForm._
 
 class Application(override val wsClient: WSClient, override val conf: Configuration, db: DB, val messagesApi: MessagesApi) extends Controller with AuthActions with I18nSupport with StrictLogging {
 
+  val AdminWithClearanceUsers: List[String] = List(
+    "alastair.jardine@guardian.co.uk",
+    "nathan.good@guardian.co.uk",
+    "pascal.honore@guardian.co.uk"
```

Comment:
Definitely be good to take these email addresses from settings that aren't in this repository.

There are several options. This can be part of the application config, you could use the Google Group Checker provided in Play-GoogleAuth, or you could use some other service to provide the list. My understanding is that other admin tools we provide use Google groups to achieve this, we can chat IRL if you like - I can point you at the right people.